runtime/lib/backends/nccl/providers/all_gather.h

```diff
@@ -0,0 +1,34 @@
+//===- all_gather.h -------------------------------------------*--- C++ -*-===//
+//
+// Copyright 2022 ByteDance Ltd. and/or its affiliates. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "brt/core/framework/op_kernel.h"
+
+namespace brt {
+namespace cuda {
+
+// send synchronously
```

Comment:
nit: send -> all gather?

Reply:
Sorry, there are some details I didn't really notice.Thanks for the advice you gave.